src/utils/bluetooth/bluetoothDataParser.ts

```diff
@@ -0,0 +1,80 @@
+import { decode } from 'base-64'
+import {
+  wifi_services_v1 as WifiServicesV1,
+  wifi_remove_v1 as WifiRemoveV1,
+  wifi_connect_v1 as WifiConnectV1,
+  add_gateway_v1 as AddGatewayV1,
+} from '@helium/proto-ble'
+import { util } from 'protobufjs'
+import { HotspotCharacteristic, FirmwareCharacteristic } from './bluetoothTypes'
+
```

Comment:
@tyler-whitman @allenan ↓↓ Do you know what this array check is here for? Is it needed still?

Reply:
I think that was to check that the response from the hotspot was well-formed, and not an error or some garbage string